index.js

```diff
@@ -149,8 +151,8 @@ function makeCachedResolver (prefix, fieldName, cache, originalFieldResolver, po
         result = await originalFieldResolver(self, arg, ctx, info)
       } else {
         // use cache to get the result
-        // note the cache function never throws error
-        result = await cache[name]({ self, arg, ctx, info })
+        // Ignore execptions, 'onError' is already in place
+        result = await cache[name]({ self, arg, ctx, info }).catch(noop)
```

Comment:
I think this should be handled differently. We should remove the `onError` in like 165 instead.

Reply:
not really agree, the `onError` in the catch block covers other user defined functions such `skip` `policy.skip` and `originalResolver` that could throw an error

Reply:
ok!